frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -157,9 +157,16 @@ local footerTextGeneratorMap = {
         local prefix = symbol_prefix[symbol_type].time
         local clock
         if footer.settings.time_format == "12" then
-            clock = os.date("%I:%M%p")
+            if os.date("%p") == "AM" then
+                -- @translators this is the time in the morning in 12-hours format (%I is the hour, %M the minute)
```

Comment:
Could you make it a proper sentence? ("This … blah blah.")

Also it's 12-hour format & 24-hour format (or maybe just `clock` instead of format?) ^_^

Reply:
You mean like this?:
This is the time in the morning in 12-hour clock (%I is the hour, %M the minute).